src/Router.tsx

```diff
@@ -44,37 +51,72 @@ const WithProvider = () => {
         },
       }}
     >
-      <Stack.Navigator initialRouteName="Home" mode="modal">
+      <Stack.Navigator initialRouteName="Home">
         <Stack.Screen
           name="Home"
           component={Home}
-          options={{ headerShown: false }}
-        />
-        <Stack.Screen
-          name="Memoir"
-          component={Memoir}
-          options={{ headerShown: false }}
-        />
-        <Stack.Screen
-          name="MemoirScreenShot"
-          component={MemoirScreenShot}
-          options={{ headerShown: false }}
-        />
-        <Stack.Screen
-          name="SettingLicence"
-          component={SettingLicence}
           options={HomeScreenOption()}
         />
         <Stack.Screen
-          name="MyPage"
-          component={MyPage}
-          options={{ headerShown: false }}
-        />
-        <Stack.Screen
-          name="SettingMemoir"
-          component={SettingMemoir}
-          options={HomeScreenOption('振り返り通知設定')}
+          name="ItemDetail"
+          component={ItemDetail}
+          options={HomeScreenOption()}
         />
+
+        <Stack.Group screenOptions={{ presentation: 'modal' }}>
```

Comment:
```
Found screens with the same name nested inside one another. Check:

Home, Home > Home

This can cause confusing behavior during navigation. Consider using unique names for each screen instead.
at node_modules/@sentry/utils/dist/instrument.js:111:20 in <anonymous>
at src/Router.tsx:5:0 in <global>
at node_modules/@react-navigation/core/src/useOnAction.tsx:135:9 in React.useCallback$argument_0
```

v6だとStack.NavigatorでラップするとWarningになるので、新しく作られたStack.Groupを使用

https://reactnavigation.org/docs/group/